apps/api-gateway/src/authz/jwt-issuer.util.spec.ts

```
@@ -0,0 +1,40 @@
+    ).toBe('https://identity.example/realms/platform/protocol/openid-connect/certs');
+    expect(() =>
+      getTrustedJwksUri('https://attacker.example/realms/platform', ['https://identity.example/realms/platform'])
+    ).toThrow('JWT issuer is not trusted');
+  });
+
+  it('fails closed when no trusted issuer configuration exists', () => {
+    expect(() => getTrustedJwtIssuers({})).toThrow('JWT_TRUSTED_ISSUERS');
+  });
+
+  it('allows HTTP only for explicit loopback issuers', () => {
+    expect(getTrustedJwtIssuers({ JWT_TRUSTED_ISSUERS: 'http://localhost:8080/realms/platform' })).toEqual([
```

ESLint

[error] There should be no linebreak after '['.

(array-bracket-newline)